holidays/countries/monaco.py

```diff
@@ -0,0 +1,100 @@
+#  python-holidays
+#  ---------------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: dr-prodigy <maurizio.montel@gmail.com> (c) 2017-2022
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/dr-prodigy/python-holidays
+#  License: MIT (see LICENSE file)
+
+from datetime import date
+
+from dateutil.easter import easter
+from dateutil.relativedelta import relativedelta as rd
+
+from holidays.constants import SUN, JAN, MAY, AUG, NOV, DEC
+from holidays.holiday_base import HolidayBase
+
+
+class Monaco(HolidayBase):
+    """
+    https://en.wikipedia.org/wiki/Public_holidays_in_Monaco
+    https://en.service-public-entreprises.gouv.mc/Employment-and-social-affairs/Employment-regulations/Leave/Public-Holidays  # noqa: E501
+    """
+
+    country = "MC"
+    special_holidays = {
+        2015: ((JAN, 7, "Jour férié [Public holiday]"),),
+    }
+
+    def _populate(self, year):
+        def _add_observed(hol_date: date) -> None:
+            if self.observed and hol_date.weekday() == SUN:
+                self[hol_date + rd(days=+1)] = f"{self[hol_date]} (Observed)"
+
+        super()._populate(year)
+
+        # New Year's Day
+        dt = date(year, JAN, 1)
+        self[dt] = "Le jour de l’An [New Year's Day]"
```

Comment:
It seems ’ is quite stubborn one. Let's replace them with ' again (line 41, 58, 87)!

Reply:
Done. Thank you!